ajax/libs/mapbox-gl/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "mapbox-gl",
+  "description": "A WebGL interactive maps library",
+  "filename": "mapbox-gl.js",
+  "license": "BSD-3-Clause",
+  "homepage": "https://www.mapbox.com/mapbox-gl-js/api/",
+  "author": "Konstantin Kaefer (mail@kkaefer.com)",
```

Comment:
The first commit of the repo is by him, and his GH profile says he still works at Mapbox. https://github.com/kkaefer